src/language/pythonLanguage.ts

```diff
@@ -52,10 +88,9 @@ export const pythonLanguage: Language = {
     biginteger: { type: 'int' },
     '': { type: 'Object', name: 'arg' },
   },
-
   defaultSnippetTemplate: `
   @{{ #lowercase }}{{ keyword }}{{ /lowercase }}('{{ expression }}')
-  def step_{{ #lowercase }}{{ keyword }}{{ /lowercase }}({{ #parameters }}{{ #seenParameter }}, {{ /seenParameter }}{{ name }}{{ /parameters }}) :
+  def step_{{ #lowercase }}{{ keyword }}{{ /lowercase }}(context, {{ #parameters }}{{ #seenParameter }}, {{ /seenParameter }}{{ name }}{{ /parameters }}) :
```

Comment:
Looks like an unrelated bugfix. This is fine, but please mention it in `CHANGELOG.md`